src/Workspaces/Core/Portable/Remote/WellKnownSynchronizationKind.cs

```diff
@@ -28,4 +28,6 @@ internal enum WellKnownSynchronizationKind : byte
     AnalyzerReference = 13,
 
     SerializableSourceText = 14,
+    FallbackAnalyzerOptions = 15,
```

Comment:
nit: put after SourceGeneratorExecutionVersionMap (fine to update versions).  As this is Solution-State date, it was intended to go there.  But i don't feel strongly.